test/test-style.js

```diff
@@ -145,29 +145,21 @@ function testStyle(filename) {
 
   if (actual !== expected) {
     hasErrors = true;
-    console.error(chalk.red(`  Style – Error on line ${jsonDiff(actual, expected)}`));
+    console.error(chalk.red(`  File : ${path.relative(process.cwd(), filename)}`));
```

Comment:
We wanted to remove the extra "File: " output, right? This looks like it is adding it back.

Reply:
Looks like a bad merge from master.  Fixes incoming!